app/models/nearby_facility.rb

```diff
@@ -7,13 +7,44 @@ class << self
     attr_writer :validate_on_load
 
     def query(params)
-      json_from_file = File.read(Rails.root.join('modules', 'va_facilities', 'nearby.json'))
-      mocked_response = JSON.parse(json_from_file)
-      return mocked_response if params[:street_address] && params[:city] && params[:state] && params[:zip]
+      if params[:street_address] && params[:city] && params[:state] && params[:zip]
+        isochrone_response = request_isochrone(params)
+        return get_facilities_in_isochrone(params, isochrone_response)
+      end
 
       NearbyFacility.none
```

Comment:
I think I would prefer to short circuit with the `none`. For example, the first line of the method would look like this:
``` if !(params[:street_address] && params[:city] && params[:state] && params[:zip])
    return NearbyFacilities.none
end

# isochrone code
```
I don't feel super strongly about it though

Reply:
Agreed. I was basing this on how `BaseFacilities` worked before your refactor simplified things, but that structure doesn't offer any benefits here. Moving `NearbyFacility.none` into an `unless` statement.